proto/kava/evmutil/v1beta1/tx.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+package kava.evmutil.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package                    = "github.com/kava-labs/kava/x/evmutil/types";
+option (gogoproto.equal_all)         = true;
+option (gogoproto.verbose_equal_all) = true;
+
+// Msg defines the evmutil Msg service.
+service Msg {
+  // ConvertCoinToERC20 defines a method for converting sdk.Coin to Kava ERC20.
+  rpc ConvertCoinToERC20(MsgConvertCoinToERC20) returns (MsgConvertCoinToERC20Response);
+
+  // ConvertERC20ToCoin defines a method for converting Kava ERC20 to sdk.Coin.
+  rpc ConvertERC20ToCoin(MsgConvertERC20ToCoin) returns (MsgConvertERC20ToCoinResponse);
+}
+
+// MsgConvertCoinToERC20 defines a conversion from sdk.Coin to Kava ERC20.
+message MsgConvertCoinToERC20 {
+  // Kava bech32 address initiating the conversion.
+  string initiator = 1;
+  // EVM 0x hex address that will receive the converted Kava ERC20 tokens.
+  string receiver = 2;
+  // Amount is the sdk.Coin amount to convert.
+  cosmos.base.v1beta1.Coin amount = 3;
+}
+
+// MsgConvertCoinToERC20Response defines the response value from Msg/ConvertCoinToERC20.
+message MsgConvertCoinToERC20Response {}
+
+// MsgConvertERC20ToCoin defines a conversion from Kava ERC20 to sdk.Coin.
+message MsgConvertERC20ToCoin {
+  // EVM 0x hex address initiating the conversion.
+  string initiator = 1;
+  // Kava bech32 address that will receive the converted sdk.Coin.
+  string receiver = 2;
```

Comment:
nit: in other modules we normally add `(cosmos_proto.scalar) = "cosmos.AddressString"` to sdk address strings. I don't think it does anything yet but probably worth keeping consistency.